```
02:35:07  CMake Error at cmake/modules/SwiftSetIfArchBitness.cmake:43 (message):
02:35:07    Unknown architecture: armv5te
02:35:07  Call Stack (most recent call first):
02:35:07    cmake/modules/SwiftHandleGybSources.cmake:110 (set_if_arch_bitness)
02:35:07    stdlib/cmake/modules/AddSwiftStdlib.cmake:998 (handle_gyb_sources)
02:35:07    stdlib/public/core/CMakeLists.txt:576 (add_swift_target_library_single)
```

Just needs one more check in `SetIfArchBitness.cmake`, but looks like `adjustClangTriple` recognizes this architecture.

--- a/cmake/modules/SwiftSetIfArchBitness.cmake
+++ b/cmake/modules/SwiftSetIfArchBitness.cmake
@@ -13,6 +13,7 @@ function(set_if_arch_bitness var_name)
      "${SIA_ARCH}" STREQUAL "x86" OR
      "${SIA_ARCH}" STREQUAL "armv4t" OR
      "${SIA_ARCH}" STREQUAL "armv5" OR
+     "${SIA_ARCH}" STREQUAL "armv5te" OR
      "${SIA_ARCH}" STREQUAL "armv6" OR
      "${SIA_ARCH}" STREQUAL "armv6m" OR
      "${SIA_ARCH}" STREQUAL "armv7" OR

